e2e/global-setup-esm/__tests__/test.js

```diff
@@ -8,7 +8,6 @@
 import os from 'os';
 import path from 'path';
 import fs from 'graceful-fs';
-import greeting from '../';
```

Comment:
Not sure. I think this should stay. The file being imported exist in parent directory. Seems like this is on purpose. Might be this import is testing resolver or something what deals with imports. There is the whole group of tests with similar import. Look at `global-setup-`, `global-teardown-` dirs.

Reply:
good call, should probably add a comment

Reply:
yeah now i looked over it, so i should add a comment above

Reply:
And please revert #12181 as well. Same case (;